src/schema/v2/me/__tests__/sendIdentityVerficationEmailMutation.test.ts

```diff
@@ -0,0 +1,96 @@
+import gql from "lib/gql"
+import { runAuthenticatedQuery, runQuery } from "schema/v2/test/utils"
+
+const identityVerificationDetails = {
+  state: "pending",
+  user_id: "id-123",
+  id: "106",
+}
+
+const sendIdentityVerificationEmailMock = jest.fn().mockReturnValue(
+  Promise.resolve({
+    id: "106",
+    state: "pending",
+    user_id: "id-123",
+  })
+)
+
+const computeMutationInput = ({
+  userID = "",
+}: {
+  userID?: string | null
+} = {}): string => {
+  const mutation = gql`
+    mutation {
+      sendIdentityVerificationEmail(
+        input: {
+          userID: ${JSON.stringify(userID)}
+        }
+      ) {
+        confirmationOrError{
+							... on IdentityVerificationEmailMutationSuccessType{
```

Comment:
**thought:** is this formatted weirdly only for me 🤔

Reply:
It looks weird on Github only. 🧐